webapp/src/test/java/com/box/l10n/mojito/service/smartling/SmartlingTestData.java

```diff
@@ -0,0 +1,98 @@
+package com.box.l10n.mojito.service.smartling;
+
+import com.box.l10n.mojito.smartling.request.*;
+import com.box.l10n.mojito.smartling.response.FilesResponse;
+import com.box.l10n.mojito.smartling.response.SourceStringsResponse;
+
+import java.util.Arrays;
+import java.util.Collections;
+
+class SmartlingTestData {
```

Comment:
this seems to be weird to have nested classes like this. Why not just have regular classes?

Reply:
This is just test data and it didn't seem worth it to make a separate file for each test case, similar to TMTestData (current class for test data)